src/sentry/models/event.py

```diff
@@ -569,7 +569,7 @@ def next_event_id(self, environments=[]):
 
         return six.text_type(result['data'][0]['event_id'])
 
-    def prev_event_id(self, environments=None):
+    def prev_event_id(self, environments=[]):
```

Comment:
You can't use [mutable types as default values](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments) in python functions.

Reply:
Ouch, thank you